std/range/package.d

```diff
@@ -6518,7 +6519,15 @@ Transversal!(RangeOfRanges, opt) transversal
     x[0] = [1, 2];
     x[1] = [3, 4];
     auto ror = transversal(x, 1);
-    assert(equal(ror, [ 2, 4 ][]));
+    assert(equal(ror, [ 2, 4 ]));
+
+    // full unzip
```

Comment:
Could you please create a new public test for this? The comment will then be displayed as tile